lib/blockscore/connection.rb

```diff
@@ -1,66 +1,57 @@
 module BlockScore
   module Connection
     MAJOR_VERSION = BlockScore::VERSION.split('.').first
-    REPO          = 'https://github.com/BlockScore/blockscore-ruby'.freeze
-    API_VERSION   = "version=#{MAJOR_VERSION}".freeze
+    REPO = 'https://github.com/BlockScore/blockscore-ruby'.freeze
+    API_VERSION = "version=#{MAJOR_VERSION}".freeze
```

Comment:
As a general rule, when I have constants or local variables on consecutive lines, all using assignment operators, I will align the assignment operators.